tools/wal-reader/main.go

```diff
@@ -28,12 +28,12 @@ func main() {
 	}
 
 	var (
-		printSeriesEntries           bool
-		printSeriesWithSampleEntries bool
+		printSeriesEntries bool
+		printSamples       bool
 	)
 
-	flag.BoolVar(&printSeriesEntries, "series-entries", true, "Print series entries")
-	flag.BoolVar(&printSeriesWithSampleEntries, "print-series-with-samples", false, "Print series information for each sample")
+	flag.BoolVar(&printSeriesEntries, "print-series", true, "Print series entries")
+	flag.BoolVar(&printSamples, "print-samples", false, "Print samples, exemplars, metadata and tombstone.")
```

Comment:
I guess it should be "tombstones"?
```suggestion
	flag.BoolVar(&printSamples, "print-samples", false, "Print samples, exemplars, metadata and tombstones.")
```